tutorial/part3/data_exploitability_pangeo.ipynb

```diff
@@ -3,7 +3,10 @@
   {
```

Comment:
<p>The explanation of stacstac could help here?</p><p>also possible other tool such as<a href="https://github.com/stac-utils/xpystac" target="_blank">https://github.com/stac-utils/xpystac</a>, odc-stac?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/pangeo-openeo-BiDS-2023/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Indeed, maybe worth to mention the alternatives. Thanks Tina, I wasn't aware of xpystac! Here an interesting discussion comparing odc-stac and stackstac: <a href="https://github.com/opendatacube/odc-stac/issues/54" target="_blank">https://github.com/opendatacube/odc-stac/issues/54</a></p>